accounting_test.go

```diff
@@ -19,8 +19,12 @@ func init() {
 	fmt.Printf("version: %s", runtime.Version())
 }
 
+func TestNewAccounting(t *testing.T) {
+
```

Comment:
Hm... This test contains empty body. Is this PR in progress?

Reply:
Oh, sorry I did add the Checks in TestFormatMoney.

Reply:
Okay. Please remove this empty test :)